packages/app/src/components/PageHistory/Revision.tsx

```diff
@@ -13,18 +13,18 @@ import styles from './Revision.module.scss';
 
 type RevisionProps = {
   revision: IRevisionHasId,
-  currentPageId: string,
-  currentPagePath: string,
   isLatestRevision: boolean,
   hasDiff: boolean,
+  currentPageId: Nullable<string> | undefined,
+  currentPagePath: Nullable<string> | undefined,
```

Comment:
I think these types no need to null and undefined, types should be only a `string`.
Because, `currentPageId` and `currentPagePath` type guard on PageHistory.tsx, is it correct?

Reply:
Yes, that's right, it's been determined from `PageHistory`. I will adjust it and remove the null checking on the Link render in Revision